docs/packing.rst

```diff
@@ -179,17 +179,19 @@ When reproducing an experiment that communicates with a server, the experiment w
 
 and use *reprozip* to trace the script execution, rather than the experiment itself. In this way, ReproZip is able to capture the local server as well, which ensures that the server will be alive at the time of the reproduction.
 
-For example, to start postgresql, then a webserver (that will run until Ctrl+C is received), then stop postgresql::
+For example, if you have an web app that uses PostgreSQL and that runs until ``Ctrl+C`` is received, you can use the following script::
 
     #!/bin/sh
 
-    /etc/init.d/postgresql start    # Start PostgreSQL
+    /etc/init.d/postgresql start        # Start PostgreSQL
 
-    trap ' ' INT                    # Don't exit the whole script on Ctrl+C
+    trap ' ' INT                        
```

Comment:
This command is the reason I'm showing the full script, shouldn't we keep the comment?